uber/model_checks.py

```diff
@@ -190,6 +190,27 @@ def printed_badge_deadline(attendee):
 
 
 @validation.Attendee
+def printed_badge_change(attendee):
+    badge_name_changes_allowed = True
+
+    # this is getting kinda messy and we probably need to rework the entire concept of "printed badge deadline".
```

Comment:
Just to clarify (why can't you edit comments on the phone, come on GitHub) I generally support making this distinction explicit, but it's also not terribly confusing as it is. Certainly I think most events will use the deadlines the way we used to. In fact, if we made Supporter an explicit badge type and checked for all preassigned badge types, it would work out of the box even if we set the config dates wrong.